Week1/homework/js-exercises/logHello.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+let helloArray = ["Hello World!",
+    "Halo, dunia! // Indonesian",
+    "Ciao, mondo! // Italian",
+    "Hola, mundo! // Spanish",
+    "Selam Dünya! // Turkish",
+    "Hallo Wereld! // Dutch",
+    "Hallo Welt! // German",
+    "안녕하세요 세계! // Korean",
+    "こんにちは世界! //Japanese",
+    "你好，世界! // Chinese"
+];
+for (let i = 0; i < helloArray.length; i++) {
+    console.log(helloArray[i]);
```

Comment:
👏 for logic but a minor indentation(spacing) issue when creating the array. You could also use an object here like:
```
const languages = {
  nederlands: 'Hallo, Wereld',
  spanish: 'Hola, mundo',
  // and so on
};
```